theme6/source/stylesheets/site.css.scss

```diff
@@ -1,159 +1,233 @@
-      width:300px;
-      justify-content: center;
-    }
-
-    footer .information ul li a{
-      height: 22px;
-      width: 192px;
-      color: #72808C;
-      font-family:'Montserrat', sans-serif;
-      font-size: 18px;
-      line-height: 21px;
-      text-decoration: none;
-      align-items: center;
-      margin-right: 50;
-
-    }
-
-    footer .information ul li{
-      display: inline-block;
-    }
-    footer .information ul{
-      margin-top: 2%;
-      text-align: center;
-    }
-
-    footer .author {
-      display:block;
-      text-align: center;
-    }
-
-    footer .author p {
-      color: #72808C;
-      font-family:'Montserrat', sans-serif;
-      font-size: 16px;
-      line-height: 18px;
-    }
-/*=============Stylesheet MOBILE==========*/
-    @media screen and(max-width:320px) {
-      .information{display: none;}
-      .logo{display:none;}
-      #twt{
-        margin-left:-20px;
-        margin-right: -20px;}
-
-        .contenedorsocial{
-          display:flex;
-        margin-left: 50px;}
-        .box-social{align-content:center;}
-        .box-social p{font-size: 14px}
-        .boxsocialmedia{
-          width: 95%;
-          display:inline-flex;
-          justify-content:space-between;
-          align-content:center;
-        }
-
-        .contenedornews{
-          margin-top: 5px;
-          display:block;
-          align-content: center;
-          width: 100%;
-        }
-        .contenedornews p{
-          font-size: 15px;
-        }
-        .contenedornews input{
-          height: 40px;
-          width: 305px;
-          border: 2px solid #D2D2D2;
-          border-radius: 2px;
-          background-color: #FBFBFB;
-        }
-        footer .author p{
-          margin-top: 35px;
-          font-size: 12px
-        }
-    }
+footer {
+  bottom: 0;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.footer-content {
+  margin: 0 40px;
+}
+
+.information {
+  display: inline-flex;
+  flex-direction: row;
+  justify-content: center;
+  margin-top: 18px;
+  text-align: center;
+  width: 100%;
+}
+
+.information ul {
+  display: flex;
+  justify-content: space-between;
+  margin-left: 0;
+  padding-left: 0;
+  width: 100%;
+}
+
+.information li {
+  display: flex;
+  justify-content: space-between;
+}
+
+.social-media-container {
+  display: flex;
+  width: 100%;
+}
+
+.social-media-box {
+  display: flex;
+  flex: 1;
+  justify-content: flex-start;
+  transform: rotateY(100px);
+}
+
+.logo {
+  align-content: center;
+  display: flex;
+  justify-content: center;
+  width: 100px;
+}
+
+.social-media {
+  color: #72808C;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 18px;
+  line-height: 21px;
+}
+
+.sm {
+  border-radius: 3px;
+  border: 2px solid #72808c;
+  height: 35px;
+  width: 93px;
+}
+
+.fb {
+  background-position: center center;
+  background-repeat: no-repeat;
+  background: url("/images/fb.svg");
+}
+
+.twt {
+  background-position: center center;
```

Comment:
Properties should be ordered background, background-position, background-repeat, margin-left, margin-right